ui.js

```diff
@@ -20,53 +20,60 @@ const STAGE_OFFLINE = 1;
 const STAGE_SEARCH = 2;
 const STAGE_COPIED = 3;
 
+const ARROW_UP = '\u001b[A';
+const ARROW_DOWN = '\u001b[B';
+const ARROW_LEFT = '\u001b[D';
+const ARROW_RIGHT = '\u001b[C';
+const ESC = '\u001b';
+const CTRL_C = '\x03';
+const RETURN = '\r';
```

Comment:
Note to self: These should be moved to https://github.com/sindresorhus/ansi-escapes